fitbenchmarking/fitbenchmarking
# tie github readme and landing page of docs

#### Description of Work

Fixes #650 


#### Testing Instructions

1. check the docs render correctly (https://fitbenchmarking.readthedocs.io/en/650-readme/)
2. check fitbenchmarking can still be installed correctly
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
